<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accomplishment</class>
 <widget class="QMainWindow" name="Accomplishment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_add_accom"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Add_Accomplish">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Previous Accomplishments: </string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="15" column="1">
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_delete"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_prev_acc_edit"/>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="delete_accomplish">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="edit_accomplish">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_new_acc_edit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Delete Accomplishment</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Add New Accomplishment: </string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Edit Accomplishment:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Previous Accomplishment Text:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>New Accomplishment Text:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="label_for_error">
      <property name="text">
       <string>Text Error:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QPushButton" name="pushButton_see_profile">
      <property name="text">
       <string>See Profile</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
